cmd/manager/main.go

```diff
@@ -281,6 +281,37 @@ func execute() {
 		},
 	}
 
+	if operator.HasRole(operator.WebhookServer, roles) {
+		autoWebhookCertificatesMngt := viper.GetBool(AutoCertificatesWebhookFlag)
+		if autoWebhookCertificatesMngt {
+			log.Info("Automatic management of the webhook certificates enabled")
+			// Ensure that all the certificates needed by the webhook server are already created
+			webhookParams := webhook.Params{
+				Namespace:                viper.GetString(OperatorNamespaceFlag),
+				SecretName:               viper.GetString(WebhookSecretFlag),
+				ServerDomainName:         viper.GetString(WebhookDomainNameFlag),
```

Comment:
I am confused as to when it might be useful to deviate from the k8s service name here (which we also have fixed in all our manifests). Can you maybe add some explanation in a comment or remove the option?